pkg/packaging/packaging.go

```diff
@@ -315,6 +316,7 @@ func (p *PackageOptions) Build(ctx context.Context, packageWriter io.Writer, tar
 		Title:                    p.Title,
 		Root:                     p.packageRoot,
 		Scripts:                  p.scriptRoot,
+		ContainerTool:            p.ContainerTool,
```

Comment:
Quick request, sorry for the late review -- could we add a default value here? e.g. above -- 

```
if p.Title == "" {
	p.Title = fmt.Sprintf("Launcher agent for %s", p.Identifier)
}

if p.ContainerTool == "" {
	p.ContainerTool = "docker"
}
```

That way when we have `cmd := exec.CommandContext(ctx, po.ContainerTool, append(args, fpmCommand...)...)` in `package_fpm` we can be sure that we are passing in "docker" rather than an empty string.

This is in reference to https://github.com/kolide/launcher/pull/1722/files#r1683280222 -- when we do packaging, we don't call package-builder via the command line, so we don't get the helpful default value from `package-builder.go` 🙂

Reply:
Looks like maintainers have access, so I added it!